src/sliding-sync.ts

```diff
@@ -82,10 +82,17 @@ export interface MSC3575SlidingSyncRequest {
     clientTimeout?: number;
 }
 
+export interface MSC4186Hero {
+    user_id: string;
+    displayname: string;
+    avatar_url: string;
```

Comment:
How can you be sure that these fields will be present?

Reply:
Mm, not sure why these weren't optional since the code was good about checking their existence. Fixed.